api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/CreateFeeDto.java

```diff
@@ -1,23 +1,27 @@
 package uk.gov.hmcts.fees2.register.api.contract.request;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
we should explicitly import, no * imports

Reply:
done